internal/templates/02-import.yaml.tmpl

```diff
@@ -8,14 +8,88 @@ commands:
 - command: sleep 10
 - command: ${KUBECTL} scale deployment crossplane -n ${CROSSPLANE_NAMESPACE} --replicas=1 --timeout 10s
 - script: ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} get deploy --no-headers -o custom-columns=":metadata.name" | grep "provider-" | xargs ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} scale deploy --replicas=1
-- script: ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} get pods --no-headers -o custom-columns=":metadata.name" | grep "provider-" | xargs ${KUBECTL} -n ${CROSSPLANE_NAMESPACE} wait pods --for=condition=Ready --timeout=60s
+- script: |
+    #!/bin/bash
+
+    function check_endpoints {
```

Comment:
Isn't it possible to define a generic `kubectl_retry` function that will retry the passed `kubectl` command together with its arguments and call it for `kubectl get endpoints` & `kubectl patch`?